rero_ils/modules/documents/mappings/v7/documents/document-v0.0.1.json

```diff
@@ -133,7 +133,8 @@
       },
       "autocomplete_title": {
         "type": "text",
-        "analyzer": "autocomplete"
+        "analyzer": "autocomplete",
+        "search_analyzer": "standard"
```

Comment:
You should use a custom analyzer with data processing (`autocomplete` without `ngram`)